HACKING.md

```diff
@@ -1,9 +1,32 @@
 # Snapcraft
 
+## Installing from pip
+
+First install a few dependencies:
+
+    sudo apt install gcc g++ make python3-venv libffi-dev libsodium-dev libapt-pkg-dev libarchive13
+
+Create and activate a new virtual environment:
+
+    python3 -m venv path/to/venv
+    source path/to/venv/bin/activate
```

Comment:
How about just "." as the path? That's easier for someone who doesn't normally use venv, others will probably know.